pkg/runtime/reconciler.go

```diff
@@ -76,6 +77,7 @@ func (r *reconciler) BindControllerManager(mgr ctrlrt.Manager) error {
 	r.kc = mgr.GetClient()
 	r.cache = ackrtcache.New(clientset, r.log)
 	r.cache.Run()
+	r.clientSet = *clientset
```

Comment:
You don't need to use the clientset. and yes, I know the cache stuff uses it, but I don't actually like the fact that the cache stuff uses the clientset, which is just syntactic sugar. See below for how to use the controller-runtime's Client interface for secrets.